src/Evolution/Systems/Burgers/Subcell/TciOnDgGrid.cpp

```diff
@@ -17,7 +17,8 @@ std::tuple<bool, evolution::dg::subcell::RdmpTciData> TciOnDgGrid::apply(
     const Mesh<1>& subcell_mesh,
     const evolution::dg::subcell::RdmpTciData& past_rdmp_tci_data,
     const evolution::dg::subcell::SubcellOptions& subcell_options,
-    double persson_exponent) {
+    double persson_exponent,
+    [[maybe_unused]] const bool stay_on_dg_continue_after_tci_failure) {
```

Comment:
sorry, I'm struggling with the `continue` on the name. Is this trying to say `stay_on_dg_after_tci_failure`?

Reply:
The intent is to indicate that even though the element has to stay on DG, the con2prim continues (or updates) after a tci failure.  If you have to ask, that means I could've chosen a clearer name though; I'll change it to `element_stays_on_dg`.